policy/datasets.yaml

```diff
@@ -1104,6 +1104,15 @@ apPipe_metadata:
     python: lsst.daf.base.PropertySet
     tables: raw
     template: ''
+apdb_marker:
+    description: >
+        Tag that indicates that data for a particular exposure have been stored in the APDB.
+        As a convenience, can be read for config information about accessing the APDB.
+    persistable: Config
+    storage: ConfigStorage
+    python: lsst.dax.ppdb.PpdbConfig
```

Comment:
Apdb ?

Reply:
The code would not have built with `ApdbConfig` at the time.